ext/bcmath/bcmath.c

```diff
@@ -174,15 +177,26 @@ PHP_FUNCTION(bcadd)
 	bc_init_num(&first);
 	bc_init_num(&second);
 	bc_init_num(&result);
-	php_str2num(&first, ZSTR_VAL(left));
-	php_str2num(&second, ZSTR_VAL(right));
+
+	if (php_str2num(&first, ZSTR_VAL(left)) == FAILURE) {
+		zend_argument_value_error(1, "bcmath function argument is not well-formed");
```

Comment:
```suggestion
		zend_argument_value_error(1, "is not well-formed");
```
This will make the error message:
> bcadd(): Argument #1 ($num1) is not well-formed

Reply:
Thanks @nikic, fixed.